src/app.js

```diff
@@ -0,0 +1,93 @@
+const express = require('express');
+const morgan = require('morgan');
+const helmet = require('helmet');
+const cors = require('cors');
+
+const axios = require('axios');
+
+require('dotenv').config();
+
+const middlewares = require('./middlewares');
+const modules = require('./modules');
+
+const app = express();
+
+app.use(morgan('dev'));
+app.use(helmet());
+app.use(cors());
+app.use(express.json());
+
+app.get('/', (_, res) => {
+  res.json({
+    ok: true
+  });
+});
+
+const getConfig = (req) => {
+  const { cookie: Cookie } = req.headers; // Retrieve the Cookie header value
+
+  return {
+    headers: {
+      Cookie
+    }
+  };
+};
+
+app.get('/style/:sId/techSpecs', async (req, res) => {
```

Comment:
For this and all following apis, we have to re-visit and connect to their DB rather than calling the api.
For now, I will approve.